service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -92,36 +96,58 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         return respond(caseDetails);
     }
 
+    private void run465(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+        var caseId = caseData.getId();
+
+        if (caseId != 1639997900244470L) {
+            throw new AssertionError(format(
+                "Migration {id = DFPL-465, case reference = %s}, expected case id 1639997900244470",
+                caseId
+            ));
+        }
+        List<String> namesToRemove = of("Stacey Halbert", "Della Phillips", "Natalie Beardsmore", "Donna Bird");
+
+        List<LegalRepresentative> legalRepresentatives = unwrapElements(caseData.getLegalRepresentatives());
+
+        List<LegalRepresentative> legalRepresentativesToRemain = legalRepresentatives.stream()
+                .filter(not(legalRepresentative -> namesToRemove.contains(legalRepresentative.getFullName())))
+                .collect(Collectors.toList());
+
+        List<Element<LegalRepresentative>> filteredLegalRepresentatives = wrapElements(legalRepresentativesToRemain);
+
+        caseDetails.getData().put("legalRepresentatives", filteredLegalRepresentatives);
+    }
+
     private void run164(CaseDetails caseDetails) {
         CaseData caseData = getCaseData(caseDetails);
         var caseId = caseData.getId();
 
         if (caseId != 1626258358022834L) {
             throw new AssertionError(format(
-                "Migration {id = DFPL-164, case reference = %s}, expected case id 1626258358022834",
-                caseId
+                    "Migration {id = DFPL-164, case reference = %s}, expected case id 1626258358022834",
+                    caseId
             ));
         }
 
         List<Element<CourtAdminDocument>> otherCourtAdminDocuments = caseData.getOtherCourtAdminDocuments();
 
         Element<CourtAdminDocument> documentElement = otherCourtAdminDocuments.stream()
-            .filter(documents ->
-                "LA Certificate.pdf".equals(documents.getValue().getDocument().getFilename()))
-            .findFirst()
-            .orElseThrow();
+                .filter(documents ->
+                        "LA Certificate.pdf".equals(documents.getValue().getDocument().getFilename()))
+                .findFirst()
+                .orElseThrow();
 
         log.info("Migration {id = DFPL-164, case reference = {}} Certificate document found",
-            caseId);
+                caseId);
         boolean removed = otherCourtAdminDocuments.remove(documentElement);
         log.info("Migration {id = DFPL-164, case reference = {}} Certificate document removed {} ",
-            caseId,
-            removed);
+                caseId,
+                removed);
         caseDetails.getData().put("otherCourtAdminDocuments", otherCourtAdminDocuments);
         caseDetails.getData().putAll(documentListService.getDocumentView(getCaseData(caseDetails)));
     }
 
-
     private void run3238(CaseDetails caseDetails) {
```

Comment:
run164() and run3238() are no longer used, so they can be removed

Reply:
I agree, just left it for reference.